.github/workflows/check-media-attachments.yml

```
@@ -0,0 +1,161 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for media attachments in HTML files
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { owner, repo } = context.repo;
+            const { number } = context.payload.pull_request;
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)